skafka/src/main/scala/com/evolutiongaming/skafka/consumer/ConsumerMetrics.scala

```diff
@@ -115,50 +118,226 @@ object ConsumerMetrics {
         labels    = LabelNames("client", "topic")
       )
     } yield { (clientId: ClientId) =>
-      new ConsumerMetrics[F] {
+      new Summaries(
+        callsCounter,
+        resultCounter,
+        latencySummary,
+        recordsSummary,
+        bytesSummary,
+        rebalancesCounter,
+        topicsLatency,
+        ageSummary,
+        clientId
+      )
+    }
+  }
 
-        def call(name: String, topic: Topic, latency: FiniteDuration, success: Boolean) = {
-          val result = if (success) "success" else "failure"
-          for {
-            _ <- latencySummary.labels(clientId, topic, name).observe(latency.toNanos.nanosToSeconds)
-            _ <- resultCounter.labels(clientId, topic, name, result).inc()
-          } yield {}
-        }
+  private final class Summaries[F[_]: Monad](
```

Comment:
why `Summaries`, when it contains `Counters`, `Summaries` and `ClientId`?

Reply:
because that's its distinguishing difference from the only other implementation – Histograms